pkg/go.mod

```diff
@@ -252,3 +253,5 @@ require (
 	lukechampine.com/frand v1.4.2 // indirect
 	sourcegraph.com/sourcegraph/appdash-data v0.0.0-20151005221446-73f23eafcf67 // indirect
 )
+
+replace github.com/petar-dambovaliev/aho-corasick => github.com/pgavlin/aho-corasick v0.0.0-20230524191112-b3bbf30f7b70
```

Comment:
Happy to pull this fork into the `pulumi` org and make it more official. The original package seems somewhat under-maintained (see e.g. https://github.com/petar-dambovaliev/aho-corasick/pull/11).

FWIW we do use the original package elsewhere, so maintaining a fork could be generally useful.